### ISSUE TITLE
Drug/Alcohol Crime Doctrine

### ISSUE THREAD

**Issue by zumbak04** [Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2#61]
**Describe your suggestion in full detail below:**
Would be nice to add Drug/Alcohol Crime religious doctrine. In vanilla, alcohol hate is tied to Islam. Untying it will create many possibilities.

**Comment by MemeWizard** [comment]
> In vanilla, alcohol hate is tied to Islam.

In vanilla it's just an additional sin in beliefs tab, isn't?

**Comment by zumbak04** [comment]
> In vanilla it's just an additional sin in beliefs tab, no?

No.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/101280759-513dae00-37e4-11eb-978a-34fb12ef82f5.png)

</details>

**Comment by MemeWizard** [comment]
> No.

From what first post was saying, I got assumption that drunkard is a criminal trait to Muslims in vanilla.
Did you actually mean "they can't get an event to become one"?

**Comment by zumbak04** [comment]
> From what first post was saying, I got assumption that drunkard is a criminal trait to Muslims in vanilla.
> Did you actually mean "they can't get an event to become one"?

No. I meant we need to create Drug/Alcohol crime doctrine that will handle limiting events, decisions instead of vanilla way where Paradox assume Islam by default limits alcohol.
